spec/support/system_description_factory.rb

```diff
@@ -352,6 +352,21 @@ def build_description(name, store, options)
       ]
     }
   EOF
+  EXAMPLE_SCOPES["groups_centos"] = <<-EOF.chomp
```

Comment:
I do not think that we need a special group for centos, since they should be linux/unix universal. I would use the group which is there and adapt the kickstart simple.cfg accordingly.